lib/msal-common/src/client/ClientCredentialClient.ts

```diff
@@ -141,6 +141,8 @@ export class ClientCredentialClient extends BaseClient {
             };
         } else {
             const requestBody = this.createTokenRequestBody(request);
+            const queryParameters = this.createTokenQueryParameters(request);
```

Comment:
Should be some common logic, this is not specific for client credentials but to all confidential client flows.